reports-backend/src/main/java/org/jboss/da/reports/api/SCMLocator.java

```diff
@@ -19,6 +22,14 @@
 
     @Getter
     @NonNull
+    private Set<Long> productIds = new HashSet<>();
+
+    @Getter
+    @NonNull
+    private Set<Long> productVersionIds = new HashSet<>();
```

Comment:
I'm sure you're right, in my limited view I just saw this as the 'request' object. Will correct.